deploy/terraform/main.tf

```diff
@@ -56,21 +59,25 @@ resource "packet_device" "tink-worker" {
   project_id       = var.project_id
 }
 
-resource "packet_device_network_type" "tink-worker-network-type" {
-  device_id = packet_device.tink-worker.id
+resource "packet_device_network_type" "tink_worker_network_type" {
+  count = var.worker_count
+
+  device_id = packet_device.tink_worker[count.index].id
   type      = "layer2-individual"
 }
 
 # Attach VLAN to provisioner
 resource "packet_port_vlan_attachment" "provisioner" {
-  device_id = packet_device.tink-provisioner.id
+  device_id = packet_device.tink_provisioner.id
```

Comment:
I ran into `Error: POST https://api.packet.net/ports/bff62cf9-5f23-49fd-b500-686e3d8730b6/assign: 422 still bonded` once while testing this.

It may make sense to add `depends_on = packet_device_network_type.tink_provisioner_network_type` here 🤔

Reply:
It sounds like a reasonable dependency to me, so let's be clear about that and write it down 👍